From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: KurumiFake <kurumiisshidowife@gmail.com>
Date: Sun, 26 Jun 2022 10:55:00 +0700
Subject: [PATCH] Revert "Expose findClass for profiler"

This reverts commit 1484ad16ef275a0ebffbe20859a37050c44b6c89.
Since Flare was dropped here, this is unneeded

diff --git a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
index 80add581d18d142835d298609ea814a0eb5244b9..41d8b95ea3d468ea845b87e69f20b2b5134a0e89 100644
--- a/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
+++ b/src/main/java/org/bukkit/plugin/java/PluginClassLoader.java
@@ -157,7 +157,6 @@ public final class PluginClassLoader extends URLClassLoader { // Spigot
         throw new ClassNotFoundException(name);
     }
 
-    public boolean _airplane_hasClass(@NotNull String name) { return this.classes.containsKey(name); } // Airplane
     @Override
     protected Class<?> findClass(String name) throws ClassNotFoundException {
         if (name.startsWith("org.bukkit.") || name.startsWith("net.minecraft.")) {
